scripts/jirautils.py

```diff
@@ -70,6 +70,9 @@ def _get_or_create_issue(project, status, labels, description, summary):
                          il=",".join(i.key for i in issues),
                          i=issue))
     else:
+        LOGGER.debug("Query: {q} Returned 0 issues. "
```

Comment:
When no issue or more than one issue is found, the query is logged, but not when there is only one. Maybe it would be better to log the query after is is defined and not in a subset of the sections of this if block.

Reply:
The logs are for unusual situations or side effects. When there are 0 issues, we create a new issue and log that. When there are >1 then something has gone wrong as there shouldn't be duplicate issues matching the criteria, so we log that as an anomaly. When exactly one issue is found no further action is required, we just return the issue key, this is the most "normal" path and IMO doesn't require a log entry.

Reply:
Logs are information, what you describe is one use case for them. The existing calls are being logged at the debug level which suggests they are for when there is something to debug, for example what happens if one issue is returned but it is not the one that was expected? How would one effectively debug that without knowing the query that was made?